File: Python PowerPoint Examples/ShapesApiDemo/deleteSlideParagraph.py
import spirecloudpowerpoint
from spirecloudpowerpoint.configuration import *

appId = "your id"
appKey = "your key"
baseUrl = "https://api.e-iceblue.cn"
configuration = Configuration(appId, appKey,baseUrl)
api= spirecloudpowerpoint.api.shapes_api.ShapesApi(configuration)

fileName = "DeleteSlideParagraph.pptx"
password = ""
storage = ""
inputFolder = "input"

slide_index = 0
shape_index = "0"
paragraph_index = 0
result =api.delete_slide_paragraph(fileName, slide_index, shape_index, paragraph_index,password=password, storage=storage, folder=inputFolder)

File: PHP PowerPoint Examples/ShapesApiDemo/deleteSlideParagraph.php
<?php

use Spire\Cloud\Powerpoint\Sdk\Api\ShapesApi;
use Spire\Cloud\Powerpoint\Sdk\Configuration;

$appId = "your id";
$appKey = "your key";
$baseUrl="https://api.e-iceblue.cn";
$configuration = new Configuration($appId, $appKey,$baseUrl);
$apiInstance = new ShapesApi($configuration);

$name = "DeleteSlideParagraph.pptx";
$password = null;
$folder = "input";
$storage = null;

$slideIndex = 0;
$shapeIndex = "0";
$paragraphs = 0;
$apiInstance->deleteSlideParagraph($name, $slideIndex, $shapeIndex, $paragraphs, $password, $folder, $storage);